packages/metro-inspector-proxy/src/Device.js

```diff
@@ -212,6 +216,35 @@ class Device {
     };
   }
 
+  // Handles cleaning up a duplicate device connection, by client-side device ID.
+  // 1. Checks if the same device is attempting to reconnect for the same app.
+  // 2. If not, close both the device and debugger socket.
+  // 3. If the debugger connection can be reused, close the device socket only.
+  //
+  // This allows users to reload the app, either as result of a crash, or manually
+  // reloading, without having to restart the debugger.
```

Comment:
nit: Use doc comment.

Reply:
I used what was used on the other event handlers, but will change this to docblock :)